tests/configs/test_data.config

```diff
@@ -320,7 +320,7 @@ params {
                 hello = "${test_data_dir}/generic/txt/hello.txt"
             }
             'cnn' {
-                 reference = "${test_data_dir}/generic/cnn/reference.cnn"
+                reference = "${test_data_dir}/generic/cnn/reference.cnn"
```

Comment:
This minor change should probably not be in here. I want to remove this file at some point anyway since we're not using it.